File: Facade/Python 3/VegNonBothRestaurant.py
from Menu import Menu
from IRestaurant import IRestaurant

class VegNonBothRestaurant(IRestaurant):
  def __init__(self):
    super().__init__()
    self.__menu = Menu("Veg Non Both Menu")
  
  def getMenus(self):
    return self.__menu

File: Facade/Javascript/VegNonBothRestaurant.js
import IRestaurant from "./IRestarutant.js";
import Menu from "./Menu.js";

class VegNonBothRestaurant extends IRestaurant {
  constructor() {
    super();
    this.menu = new Menu("Veg Non Both Menu");
  }

  getMenus() {
    return this.menu;
  }
}

export default VegNonBothRestaurant;
